generate-docs.js

```diff
@@ -0,0 +1,64 @@
+//@ts-check
+
+/*
+
+This file is used to generate the sync the list.json file with all the props and methods of a Class.
+
+How to...
+
+1. First install the dependency `npm i node-fetch` (`npm i` or `npm install` won't work, as node-fetch is not there in the package.json)
+2. The json file should be available in `./docs/list.json`
+3. Then: `node generate-docs`
+4. This will generate a list.json file in the same dir you can merge the list.json with the ./docs/list.json
+
+*/
+const fetch = require('node-fetch')
+const fs = require('fs')
+let base = 'http://threejs.org/docs/'
+// @ts-ignore
+let list = JSON.parse(fs.readFileSync('./docs/list.json'))
+
+const timer = ms => new Promise(res => setTimeout(res, ms))
+
+async function task(i) { 
+    //3000ms can be variable according to your internet speed and performance (lowering this value may cause connection timeout error)
+    await timer(3000);
```

Comment:
maybe use `fs.readFileSync()` to avoid doing this? All documentation files are present in the repo